src/jsonUtils/nodeImpl/graphics/curvePoint.js

```diff
@@ -0,0 +1,38 @@
+import type { Point } from './types';
+
+// SJCurvePoint types copied from sketchapp-json-flow-types (they're not exported)
+type PointString = string;
+type CurveMode = 0 | 1 | 2 | 3 | 4;
+export type SJCurvePoint = {|
```

Comment:
I just published sketchapp-json-flow-types@0.3.6 which exposes SJCurvePoint and SJCurveMode

Reply:
Updated!